solutions/devsprint-leonardo-santos-1/FinanceAppTests/Service/FinanceServiceTest.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  FinanceServiceTest.swift
+//  FinanceAppTests
+//
+//  Created by Leonardo Cunha on 21/06/22.
+//
+
+import XCTest
+@testable import FinanceApp
+
+final class FinanceServiceTests: XCTestCase {
+    private let networkSpy = NetworkClientSpy()
+    private lazy var sut = FinanceService(networkClient: networkSpy)
+    
+    // MARK: fetchContactList
+    func test_fetchContactList_whenDecondeIsSuccessful_objectShouldMatchMockedContactList() {
+        networkSpy.performRequestCompletionDataToBeReturned = contactListData
+        let mockedContactList = [Contact.fixture(name: "Ronald Robertson",
+                                                 phone: "+55 (11) 99999-9999"),
+                                 Contact.fixture(name: "Johnny Watson",
+                                                 phone: "+55 (11) 99999-9999")]
```

Comment:
Duas grandes vantagens da fixture é: 

1. por ser uma função estática, podemos remover o nome da classe (já que sabemos que será um array de contacts), então de `Contact.fixture()` podemos fazer `.fixture()`
2. Já que os parametros das fixtures não estão sendo utilizados, vale a pena remove-los. O que acha?

```suggestion
        let mockedContactList = [.fixture(),  .fixture()]
```